src/Command/RunCommand.php

```diff
@@ -67,6 +67,15 @@
  */
 final class RunCommand extends BaseCommand
 {
+    /** @var string */
+    private const OPTION_COVERAGE = 'coverage';
+
+    /** @var string */
+    private const OPTION_COVERAGE_XML_DIR = 'coverage-xml-dir';
+
+    /** @var string */
+    private const OPTION_JUNIT_LOG_FILE = 'junit-log-file';
```

Comment:
These could follow the style of PHPUnit: `--coverage-xml` `--log-junit`

What do you think?

Reply:
I'm implementing this according to description in ticket https://github.com/infection/infection/issues/1377. I could change this without any problem if you need. Your project, your rules :wink:

Reply:
What do you think will be most sensible, to use some original keys, or use the same keys PHPUnit uses to generate the very reports?

Reply:
It makes sense to use the same parameter names as in phpunit. I'm going to change this.

Reply:
That's a good point about the project, but that's _your_ pull request. Therefore:
- It's _your_ choice how it's going to be. If you make it good, the entire project will be better. E.g. assuming you're using Infection, if you'd make it a poo, you'll be using a poo. Let's focus on not making it worse, but better, together.
- I'm not an Einstein-like genius, I can't know the true reasons you want to use this or that name here, so if you can say there's a reason to do this that way or another, then it could be just so. I will stand corrected then.

Reply:
@sanmai  I will create new issue for this.